hooks/post-checkout

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
```

Comment:
Should we include these from a repo central location?

Reply:
I could copy over `docker/base/sbin/strict-mode.sh` from our $work project here -- originally that file was in another OSS project of mine, so there is no problem copying it.

Reply:
Yeah, I'd like if we could de duplicate and centralize here. But its not a blocker for me. your call.